app/src/main/java/org/mozilla/fenix/logins/SavedLoginSiteInfoFragment.kt

```diff
@@ -29,11 +32,25 @@ class SavedLoginSiteInfoFragment : Fragment(R.layout.fragment_saved_login_site_i
     override fun onPause() {
         // If we pause this fragment, we want to pop users back to reauth
         if (findNavController().currentDestination?.id != R.id.savedLoginsFragment) {
+            activity?.window?.clearFlags(WindowManager.LayoutParams.FLAG_SECURE)
             findNavController().popBackStack(R.id.loginsFragment, false)
         }
         super.onPause()
     }
 
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = super.onCreateView(inflater, container, savedInstanceState)
+        activity?.window?.setFlags(
```

Comment:
Could this be in onResume to match with onPause?

Reply:
ya updating